src/pybind/mgr/cephadm/templates/services/nvmeof/ceph-nvmeof.conf.j2

```diff
@@ -16,6 +16,9 @@ omap_file_lock_duration = {{ spec.omap_file_lock_duration }}
 omap_file_lock_retries = {{ spec.omap_file_lock_retries }}
 omap_file_lock_retry_sleep_interval = {{ spec.omap_file_lock_retry_sleep_interval }}
 omap_file_update_reloads = {{ spec.omap_file_update_reloads }}
+allowed_consecutive_spdk_ping_failures = {{ spec.allowed_consecutive_spdk_ping_failures }}
+spdk_ping_interval_in_seconds = {{ spec.spdk_ping_interval_in_seconds }}
+ping_spdk_under_lock = {{ spec.ping_spdk_under_lock }}
```

Comment:
is the `quay.io/ceph/nvmeof:1.0.0` image we currently have as the default able to handle these settings? If not, we may want to update the default image to one that does.

Reply:
@adk3798 having extra fields in the conf file is harmless. If the gateway doesn't know about them it wouldn't try to read them. The opposite situation is the problem, where the gateway tries to read a field which is not in the conf file.